presidio-anonymizer/tests/test_anonymizer_engine.py

```diff
@@ -157,6 +157,25 @@ def test_given_several_results_then_we_filter_them_and_get_correct_mocked_result
     assert result.items[0].text == "text"
 
 
+def test_given_sorted_analyzer_results_merge_entities_separated_by_white_space():
```

Comment:
Could you please add a few more test cases:
- For multiple spaces
- For two entities that are not from the same type but have a whitespace between them
- For two entities of the same type with a character different from whitespace between them

Consider using `@pytest.mark.parametrize`

Reply:
I have added the test cases, but I'm not sure if the way I have currently written them is very elegant. Do let me know if I need to modify. (Commit [8620534](https://github.com/microsoft/presidio/pull/1092/commits/86205342be1e9ec909a78e5bc2e9033cf48c0e4b))

Reply:
Looks good, thanks!